src/vec/mod.rs

```diff
@@ -408,7 +415,7 @@ impl<T, S: Storage> VecInner<T, S> {
     }
 
     /// Returns the maximum number of elements the vector can hold.
-    pub fn capacity(&self) -> usize {
+    pub fn storage_capacity(&self) -> usize {
```

Comment:
@sosthene-nitrokey, why can't this use the same name?

Reply:
> so for Vec<T, N> both apply, so it'd give ambiguity errors if the user wrote vec.capacity()

It wouldn't even allow  getting to that point. The compiler would complain about duplicate definitions with the same name.

We could implement `fn capacity` on `Vec` and `VecView` independently, but then it would make it unusable in a generic context over the storage.